### PR TITLE
Added support for textColor in DatePickerIOS

### PR DESCRIPTION
**Pull request by nitrogenlabs**
This update allows the text/label color of the DatePickerIOS to be updated via an attribute called, "textColor". Will solve the problem found here:

https://github.com/facebook/react-native/issues/4473

### CODE REVIEW

File: React/Views/RCTDatePickerManager.m [hunk 1]
```diff
@@ -41,5 +41,9 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(onChange, RCTBubblingEventBlock)
 RCT_REMAP_VIEW_PROPERTY(mode, datePickerMode, UIDatePickerMode)
 RCT_REMAP_VIEW_PROPERTY(timeZoneOffsetInMinutes, timeZone, NSTimeZone)
-
+RCT_CUSTOM_VIEW_PROPERTY(textColor, UIColor, RCTDatePicker)
+{
+  UIColor *textColor = json ? [[UIColor alloc] initWithCGColor:[RCTConvert CGColor:json]] : [UIColor blackColor];
+  [view setValue:textColor forKeyPath:@"textColor"];
```
Comment (javache) [inline React/Views/RCTDatePickerManager.m]: This is equivalent to using private APIs, which is reason for rejection in the App Store.

Comment (nitrogenlabs) [inline React/Views/RCTDatePickerManager.m]: Yes, this would not be good. Thanks!

**Comment by hramos** [comment]
@nitrogenlabs do you still plan on addressing the feedback here?

**Comment by nitrogenlabs** [comment]
Closing due to use of private APIs.

**Comment by pinguinjkeke** [comment]
What's a private API here?

**Pull request closed without merging**